deep-learning/_posts/2017-02-28-you-only-look-once.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: review
+title:  "You Only Look Once: Unified, Real-Time Object Detection (YOLO)"
+author: "Frédéric Branchaud-Charron"
+cite:
+    authors: "J. Redmon, S. Divvala, R. Girshick,& A. Farhadi"
+    title:   "You only look once: Unified, real-time object detection"
+    venue:   "Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition (pp. 779-788)"
+pdf: "https://arxiv.org/pdf/1506.02640.pdf"
+tags:   cnn deep-learning object-detection bounding-boxes
+---
+
+## Summary
+
+**YOLO** is an unified solution to the object detection problem. Other approches like **R-CNN** are using a two step process *(localization and classification)* where **YOLO** is a one step process. Also, **Fast YOLO** (or tiny-yolo) is achieving 155 fps where **Faster R-CNN** is doing 18 fps.
```

Comment:
an unified --> a unified
Sorry, I'm a grammar nazi. Source: https://www.grammarly.com/blog/indefinite-articles-a-and-an/